README-PL.md

```diff
@@ -150,10 +150,9 @@ Linki wyszukiwania które wskazują na kwestie skierowane dla początkujących.
 
 ## Inicjatywy kontrybucji w otwartym oprogramowaniu
 - [Up For Grabs](https://up-for-grabs.net/) - Zawiera projekty z zagadnieniami dla początkujących.
-- [First Timers Only](https://www.firsttimersonly.com/) - Lista błędów oznaczonych dla tych co dopiero zaczynają *(ang. first timers only)*
 - [First Contributions](https://firstcontributions.github.io/) - Stwórz swoją pierwszą kontrybucje do projektu Open Source w 5 minut. Narzedzie i poradnik stworzony, aby pomóc początkującym zacząć przygodę z kontrybuowaniem. [Tutaj ANG](https://github.com/firstcontributions/first-contributions) [Tutaj PL](https://github.com/firstcontributions/first-contributions/blob/master/translations/README.pl.md) jest to kod źródłowy GitHuba, który pozwala wykonać swoją pierwszą kontrybucję do repozytorium.
 - [Hacktoberfest](https://hacktoberfest.digitalocean.com/) - Program zachęcający do kontrybuowania w projektach Open Source. Zarób prezenty takie jak koszulki czy naklejki za co najmniej 4 Pull Requesty w Październiku.
 - [24 Pull Requests](https://24pullrequests.com) - jest to projekt stworzony do promowania współpracy Open Source w Grudniu. 
 
 ## Licencja
-<a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://licensebuttons.net/l/by-sa/4.0/88x31.png" /></a><br />Ta praca jest licencjonowana za pomocą <a rel="license" href="https://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.
```

Comment:
This can be kept and added back in.

Reply:
e8c131ea1774d90dd3a30cabd529c5f16d1ce230 done